### PR TITLE
AT-56 Dashboard Charts

### PR DESCRIPTION
**Pull request by FractalFraction**
Co-authored-by b7rch4 james.birchall4@btinternet.com

https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-36

Problem: The dashboard page only contained text elements, wanted to add some visual fidelity to the page using charts. 

Solution: Added doughnut and bar charts from chartjs to visualise the gender, background and challenge ratio data.

### CODE REVIEW

File: src/components/Dashboard/buildData.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const buildLabelArray = (data) => {
```
Comment (dearshrewdwit) [inline src/components/Dashboard/buildData.js:1]: Simplify

```js
const buildLabelArray = (data) => data.map(obj => obj.type );
const buildNumberArray = (data, attr) => data.map(obj => parseInt(obj[attr]));
```

File: src/components/Dashboard/buildData.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+const buildNumberArray = (data, attr) => {
+  const numbersArr = data.map(obj => parseInt(obj[attr]));
+  return numbersArr;
+}
+
+const buildData = (data, attr) => {
+
+  const dataObj = {
```
Comment (dearshrewdwit) [inline src/components/Dashboard/buildData.js:13]: Simplify: 
```js
return dataObj = {
```

Comment (dearshrewdwit) [reply src/components/Dashboard/buildData.js:13]: And then remove the `return` below

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/buildData.js (hunk 1 above)
Comment (FractalFraction) [reply src/components/Dashboard/buildData.js]: Implemented changes.

Comment (FractalFraction) [reply src/components/Dashboard/buildData.js]: Made changes

Comment (dearshrewdwit) [reply src/components/Dashboard/buildData.js]: Effected changes did you?

**Review by dearshrewdwit — APPROVED** [review]

**Review by richardtully — APPROVED** [review]

**Review by Reginmaru — APPROVED** [review]

**Pull request merged**